crates/wasmtime/src/store.rs

```diff
@@ -131,7 +134,25 @@ impl Hash for HostInfoKey {
 
 impl Store {
     /// Creates a new store to be associated with the given [`Engine`].
```

Comment:
Could the documentation here be expanded to indicate that the created `Store` has no limits on usage of memory and such in wasm modules?